CSharp/DevVmPowershell/Helpers/Implementations/InstanceSettingsHelper.cs

```diff
@@ -48,33 +74,72 @@ public bool UpdateInstanceSettingValue(string name, string section, string newVa
 				{
 					newValue = string.Empty;
 				}
-				using (IInstanceSettingManager instanceSettingManager = ServiceFactory.CreateProxy<IInstanceSettingManager>())
+
+				HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+				string queryUrl = $"Relativity.REST/api/Relativity.Objects/workspace/{Constants.EDDS_WORKSPACE_ARTIFACT_ID}/object/query";
```

Comment:
I would move all the URL's to the Constants class so they are in one place for easy updates in future.

- [ ] Move URL to Constants class